source/Octopus.Tentacle/Kubernetes/Scripts/RunningKubernetesJob.cs

```diff
@@ -345,8 +345,17 @@ async Task CreateJob(IScriptLogWriter writer, string? imagePullSecretName, Cance
                                         new(EnvironmentVariables.TentacleVersion, Environment.GetEnvironmentVariable(EnvironmentVariables.TentacleVersion)),
                                         new(EnvironmentVariables.TentacleCertificateSignatureAlgorithm, Environment.GetEnvironmentVariable(EnvironmentVariables.TentacleCertificateSignatureAlgorithm)),
                                         new("OCTOPUS_RUNNING_IN_CONTAINER", "Y")
-                                        
+
                                         //We intentionally exclude setting "TentacleJournal" since it doesn't make sense to keep a Deployment Journal for Kubernetes deployments
+                                    },
+                                    Resources = new V1ResourceRequirements
+                                    {
+                                        //set resource requests to be quite low for now as the jobs tend to run fairly quickly
```

Comment:
I'm just running a long script to see if we can get some idea of memory usage

Reply:
![CleanShot 2024-02-08 at 12 05 45@2x](https://github.com/OctopusDeploy/OctopusTentacle/assets/2888279/798f43cb-c1ae-4340-a9a4-79032baf359b)

A simple script on my Mac uses 126Mi

My Tentacle uses ~160Mi

How does it look on your cluster @APErebus ?

Reply:
Btw I had to install metrics server, and modify the args on the Deployment to add `--kubelet-insecure-tls`

https://github.com/kubernetes-sigs/metrics-server?tab=readme-ov-file#configuration

Reply:
@flin-8 Let's chat on zoom and report back here :)

Reply:
@flin-8 The most I got was ~50MiB of ram used for a plain `sleep 300` job

I think we can bump the Agent container to `150MiB`, but I think the requests we have here are a good starting point for the job

Reply:
Given that most scripts might run a CLI of some sort, maybe we should still increase the request to 100MB, what do you think?

Reply:
I'm happy with that. We can always tweak if we need

Reply:
Updated to `100Mi` in [51df9ce](https://github.com/OctopusDeploy/OctopusTentacle/pull/797/commits/51df9ce2619a5444b78e7cfb35229f27fc494c0d)